mito-ds/monorepo
# mitosheet: add helper packages, deploy to test-pypi

# Description

This PR implements this spec: https://www.notion.so/trymito/Easy-Pro-activation-for-Teams-3a9b4fb75c6642029ef63046299680d3

Also removes the mitosheet-private package (check how we're checking for no display of upgrade screen now), as we can now give different (and easier) instructions for installing Mito Pro.

**One question**: do we want to rename these? So they don't show up so easily in the mitosheet search on pypi. If so, what do we want to rename them too?

# Testing

1. Make sure pro is turned off in user json.
2. Run `python3 -m pip install --index-url https://test.pypi.org/simple/ mitosheet-helper-pro --upgrade`
3. Check that pro is active

Do the same for `python3 -m pip install --index-url https://test.pypi.org/simple/ mitosheet-helper-private --upgrade`

# Documentation

Update the pro docs for installing!
